data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -516,7 +603,7 @@ class ApiServerClient {
     logger.debug('Get resource with opts: ', configName);
     return Promise.try(() => this.init())
       .then(() => apiserver.api[CONST.APISERVER.CONFIG_MAP.API_VERSION]
-        .namespaces(CONST.APISERVER.NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE](configName).get())
+        .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE](configName).get())
```

Comment:
same as above. Why default namespace alone?

Reply:
This is not part of this PR. Only the constant name was changed. Hence let's keep it out of this PR. But I agree with you. :)